src/tests/searchParameters/search_by_period/fixtures/Observation/observation1.json

```diff
@@ -0,0 +1,81 @@
+{
```

Comment:
We should add multiple observations to represent all kinds of data in the DB with respect to datetimes for observation resource.